src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -386,6 +386,9 @@ object SymDenotations {
     /** Is symbol a primitive value class? */
     def isPrimitiveValueClass(implicit ctx: Context) = defn.ScalaValueClasses contains symbol
 
+    /** Is symbol a primitive value class? */
+    def isNumericValueClass(implicit ctx: Context) = defn.ScalaNumericValueClasses contains symbol
```

Comment:
Naming of this method is not in line with comment. As comment says nothing about class being numeric, and name of the method does not say anything about class being primitive. Methods tests for both of those.